tasks/otel_agent.py

```diff
@@ -24,8 +24,8 @@ def build(ctx):
         os.remove(BIN_PATH)
 
     env = {"GO111MODULE": "on"}
-    build_tags = ['otlp']
-    ldflags = get_version_ldflags(ctx, major_version='7')
+    build_tags = ["otlp"]
+    ldflags = get_version_ldflags(ctx)
```

Comment:
major_version is declared by default inside `get_version_ldflags`
Unless we are setting this differently it is probably safer to leave the default so it doesn't get missed in a major version upgrade release